cabal-install-solver/src/Distribution/Solver/Types/PkgConfigDb.hs

```diff
@@ -79,6 +86,13 @@ readPkgConfigDb verbosity progdb = handle ioErrorHandler $ do
 
     ioErrorHandler :: IOException -> IO PkgConfigDb
     ioErrorHandler e = noPkgConfig (show e)
+    getIndividualVersion pkgConfig pkg = do
```

Comment:
I think this function would be clearer with a type signature.

Reply:
done.